tests/tests.py

```diff
@@ -184,6 +184,7 @@ def __init__(self):
                 else:
                     self.CI_JIRA_USER_PASSWORD = 'sd4s3dgec5fhg4tfsds3434'
 
+                self.CI_JIRA_ADMIN_EMAIL_DOMAINE_NAME = os.environ.get('CI_JIRA_ADMIN_EMAIL_DOMAINE_NAME', 'ssbarnea.33mail.com')
```

Comment:
:D The french in me... I changed it. Will push when found a solution to what I last reported on #366